tests/rust/tcp-test/close/mod.rs

```diff
@@ -7,7 +7,10 @@
 
 use anyhow::Result;
 use demikernel::{
-    runtime::fail::Fail,
+    runtime::{
+        fail::Fail,
+        types::demi_opcode_t::DEMI_OPC_CLOSE,
```

Comment:
Instead of including this, could you use `demi_opcode_t::DEMI_OPC_CLOSE` in the code?